README.md

```diff
@@ -75,7 +75,7 @@ export class HealthController {
 
 ```
 
-If everything is set up correctly, you can access the healthcheck on `http://localhost:3000/health`.
+If everything is set up correctly, you can access the health check on `http://localhost:3000/health`.
```

Comment:
Uhm why are you removing the space between "health check" in line 38, but here adding a space?

Reply:
🙈 fixed. I really don't know why 🤣. 
Thanks for this review.